src/Toolkit/Str.php

```diff
@@ -972,7 +980,7 @@ public static function safeTemplate(
 	public static function short(
 		string $string = null,
 		int $length = 0,
-		string $appendix = '…'
+		string|false $appendix = '…'
```

Comment:
Was `false` as the argument already supported or used before?

Reply:
Yes, not in tests but in `Str::slug()`: https://github.com/getkirby/kirby/blob/v4/develop/src/Toolkit/Str.php#L1114

Reply:
Should we really officially support this though by adding the type hint for `false`? If `false` is passed to an argument that is hinted for `string`, the `false` value is converted to an empty string. This can be made explicit in the calling code (here: `Str::short()`) by passing an empty string instead of `false` in the first place.

Reply:
@distantnative Ping

Reply:
Idk, I'm indifferent, don't have any issue with supporting `false` either